packages/node_modules/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -100,7 +100,9 @@ registerInternalPlugin('calendar', Calendar, {
             ctx.transform('decryptTextProp', 'webexURL', object.encryptionKeyUrl, object),
             ctx.transform('decryptTextProp', 'spaceMeetURL', object.encryptionKeyUrl, object),
             ctx.transform('decryptTextProp', 'spaceURI', object.encryptionKeyUrl, object),
-            ctx.transform('decryptTextProp', 'spaceURL', object.encryptionKeyUrl, object)
+            ctx.transform('decryptTextProp', 'spaceURL', object.encryptionKeyUrl, object),
+            ctx.transform('decryptTextProp', 'meetingJoinURI', object.encryptionKeyUrl, object.meetingJoinInfo),
+            ctx.transform('decryptTextProp', 'meetingJoinURL', object.encryptionKeyUrl, object.meetingJoinInfo)
```

Comment:
To confirm, it seems as though you are decrypting a new field. Is there any way that we could add tests to this to validate that it works?